### ISSUE TITLE
Regarding the run train.py, at the end of each epoch, the call test.py computes the mAP of the best.pt

### ISSUE THREAD

**Issue by Wanghe1997** [ultralytics/yolov5#7095]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I use yolov5-5.0 code。At the end of each epoch during the training process, the mAP of the best.pt is calculated, and I know that the test.py is called. My questions is：
The conf_thres and iou_thres values are based on the values in the code "Test best.pt" in the train.py?

> 

        logger.info('%g epochs completed in %.3f hours.\n' % (epoch - start_epoch + 1, (time.time() - t0) / 3600))
        if opt.data.endswith('coco.yaml') and nc == 80:  # if COCO
            for m in (last, best) if best.exists() else (last):  # speed, mAP tests
                results, _, _ = test.test(opt.data,
                                          batch_size=batch_size * 2,
                                          imgsz=imgsz_test,
                                          conf_thres=0.001,
                                          iou_thres=0.7,
                                          model=attempt_load(m, device).half(),
                                          single_cls=opt.single_cls,
                                          dataloader=testloader,
                                          save_dir=save_dir,
                                          save_json=True,
                                          plots=False,
                                          is_coco=is_coco)

Or is it based on the default parameters in the __main__ in the test.py?

> 

    parser.add_argument('--conf-thres', type=float, default=0.001, help='object confidence threshold')
    parser.add_argument('--iou-thres', type=float, default=0.6, help='IOU threshold for NMS')

In other words, when calculating the mAP of best.pt during training, iou_thres use 0.7 or 0.6? Thanks！

**Comment by glenn-jocher** [comment]
@Wanghe1997 val.run() is called here during training:
https://github.com/ultralytics/yolov5/blob/a2d617ece94dcd8c9bc205ea70f1223c84fdbe3a/train.py#L365-L380
